zbus_macros/iface.txt

```diff
@@ -0,0 +1,6061 @@
+mod iface {
```

Comment:
I'm assuming this whole file was some kind of mistake? I'll remove it in my rebase.